backend/src/layer/data/swka_parser/html_parser.rs

```diff
@@ -133,6 +133,8 @@ lazy_static! {
     static ref VOLUME_REGEX: Regex = Regex::new(r"([0-9]*),([0-9]{2}) l").expect(REGEX_PARSE_E_MSG);
 
     static ref ID_REGEX: Regex = Regex::new(r"[0-9]{18,}").expect(REGEX_PARSE_E_MSG);
+
+    static ref POULTRY_REGEX: Regex = Regex::new(r"(Pute|(G|g)eflügel|H(ü|ä|u|a)hn)").expect(REGEX_PARSE_E_MSG);
```

Comment:
I think we should add the regex i flag to ignore case. This way we have no problems if these keywords appear in the middle of a sentence.
https://docs.rs/regex/latest/regex/struct.RegexBuilder.html#method.case_insensitive